app/views/idv/inherited_proofing/get_started.html.erb

```diff
@@ -31,7 +31,10 @@
           },
           {
             url: 'https://www.va.gov/resources/managing-your-vagov-profile/',
```

Comment:
The text is agnostic but the links are still tied to the VA information, causing confusion for users.

Reply:
Good catch. Forest through the trees.

Reply:
This card is about the I18n files/keys, but I am going to try to fix this.